src/ui/rendering/handlebarsrenderer.js

```diff
@@ -194,8 +194,24 @@ export default class HandlebarsRenderer extends Renderer {
     });
 
     this.registerHelper('processTranslation', function (options) {
+      const pluralizationInfo = {};
+      const interpolationParams = {};
       let { phrase, count } = options.hash;
-      return TranslationProcessor.process(phrase, options.hash, count);
+
+      Object.entries(options.hash).forEach(([key, value]) => {
+        if (key === 'locale') {
+          pluralizationInfo['locale'] = value;
+        } else if (key.length > 10 && key.substring(0, 10) === 'pluralForm') {
```

Comment:
Is `pluralForm` a valid param name? If so, does the check need to be `key.length >= 10`? To make things a bit more readable, I'd suggest making a local variable called `isPluralFormParam` (or some such name). If we use that here, I think it makes the code a bit easier to follow. People could easily see that you treat pluralForm params and locale params specially. If they care, they can dig into how we determine it's a pluralForm param.

Reply:
No, `pluralForm` is not a valid param name because if pluralization is being used, the key `pluralForm0` will represent the singular form, and the key `pluralForm1` will represent the pluralForm (for languages such as english). 

I'll add `isPluralFormParam` as you suggested!